src/Shared/Controls/Settings/MusicSources/ConnectNew/Ipfs/FolderSelector.xaml

```diff
@@ -0,0 +1,100 @@
+﻿<Page
+    x:Class="StrixMusic.Controls.Settings.MusicSources.ConnectNew.Ipfs.FolderSelector"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:behaviors="using:OwlCore.WinUI.Behaviors"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    xmlns:core="using:Microsoft.Xaml.Interactions.Core"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:interactivity="using:Microsoft.Xaml.Interactivity"
+    xmlns:labs="using:CommunityToolkit.Labs.WinUI"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:storage="using:StrixMusic.Controls.Storage"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+    mc:Ignorable="d">
+
+    <Grid
+        MaxWidth="1000"
+        Margin="20"
+        RowSpacing="20">
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+        </Grid.RowDefinitions>
+
+        <!--<StackPanel Visibility="{x:Bind InvertBoolToVisibility(AuthenticateCommand.ExecutionTask.IsFaulted), Mode=OneWay, FallbackValue=Visible}">
+            <TextBlock
+                HorizontalAlignment="Center"
+                FontSize="20"
+                Text="Select a folder"
+                TextAlignment="Center"
+                TextWrapping="WrapWholeWords" />
+
+            <TextBlock
+                HorizontalAlignment="Center"
+                FontSize="12"
+                Opacity="0.5"
+                Text="{x:Bind Settings.UserDisplayName, Mode=OneWay}"
+                TextAlignment="Center"
+                TextWrapping="WrapWholeWords" />
+        </StackPanel>-->
+
+        <!--<StackPanel Visibility="{x:Bind BoolToVisibility(AuthenticateCommand.ExecutionTask.IsFaulted), Mode=OneWay, FallbackValue=Collapsed}">
+            <TextBlock
+                HorizontalAlignment="Center"
+                FontSize="20"
+                Text="Something went wrong"
+                TextAlignment="Center"
+                TextWrapping="WrapWholeWords" />
+
+            <TextBlock
+                HorizontalAlignment="Center"
+                FontSize="12"
+                Foreground="Red"
+                Opacity="0.5"
+                Text="{x:Bind AuthenticateCommand.ExecutionTask.Exception.Message, FallbackValue='', Mode=OneWay}"
+                TextAlignment="Center"
+                TextWrapping="WrapWholeWords" />
+        </StackPanel>-->
+
+        <StackPanel Grid.Row="1" Spacing="12">
+            <Image MaxWidth="250" Source="ms-appx:///Assets/Cores/OneDrive/login.svg" />
```

Comment:
We should use a different svg for the Ipfs selector. Pick a good one from https://undraw.co/ with the color set to Strix purple: #BA27A9
```suggestion
            <Image MaxWidth="250" Source="ms-appx:///Assets/Cores/Ipfs/login.svg" />
```

Reply:
Resolved with this icon:
![image](https://github.com/Arlodotexe/strix-music/assets/15306909/3942a53e-4cd0-4dfc-973d-ef78e740761f)